Atomic/storage.py

```diff
@@ -165,6 +167,14 @@ def modify(self):
                 self._remove_devices(get_dss_devs(self.dss_conf), only_unused=True)
             if self.args.driver:
                 self._driver(self.args.driver)
+            if (self.args.rootfs and not self.args.lvname):
```

Comment:
@rhatdan @baude Since both the options (`--rootfs`, `--lvname`) are to be used together, I think an integration test would suit better.